### PR TITLE
[DOCFIX] Update spark documentation

### PR DESCRIPTION
**Pull request by ZacBlanco**

### CODE REVIEW

File: docs/en/compute/Spark.md [hunk 1]
```diff
@@ -90,19 +90,19 @@ of the input file `Input`.
...
+Alluxio transparently fetches data from the under storage system,
 given the exact path.
 For this section, HDFS is used as an example of a distributed under storage
 system.
 
 Put a file `Input_HDFS` into HDFS:
 
 ```console
-$ hdfs dfs -put -f ${ALLUXIO_HOME}/LICENSE hdfs://localhost:9000/alluxio/Input_HDFS
+$ hdfs dfs -copyFromLocal -f ${ALLUXIO_HOME}/LICENSE hdfs://localhost:9000/alluxio/Input_HDFS
```
Comment (ZacBlanco) [inline docs/en/compute/Spark.md:101]: `-put` did not work for me. Data never actually was loaded into my hdfs cluster when I was testing. `-copyFromLocal` worked though.

**Review by LuQQiu — APPROVED** [review]
LGTM, thanks for fixing the doc!

**Comment by LuQQiu** [comment]
alluxio-bot, merge this please

**Comment by LuQQiu** [comment]
alluxio-bot, cherry-pick this to branch-2.5 please

**Pull request merged**